specs/DesignDiscussions/InputLabels.md

```diff
@@ -32,76 +32,15 @@ Consider the following card:
 In that card, the label for the input is “Please enter your name:”, however the card renderer has no way to determine that this is the case. We’ve run into a few cases where this is an issue:
  - Accessibility: The input will be read by screen readers as “Last, First”, rather than “Please enter your name” making it more difficult for screen reader users to use the card. 
 
- - Validation indicators: We have considered allowing card authors to mark inputs as optional or required. One way to indicate that visually would be to put and indicator next to the input’s label. This requires knowing which text is the label.
+ - Validation indicators: As part of input validation, we are allowing card authors to mark inputs as optional or required. One way to indicate that visually would be to put and indicator next to the input’s label. This requires knowing which text is the label.
 
```

Comment:
March 27th: Matt, Paul, David and Alberto agreed on labels will get the benefit of isRequired markers, stand alone text blocks will not have markers